test/mocks/categories.mocks.dart

```diff
@@ -9,7 +9,35 @@ final mockAnimalCategory = ApiCategory(
 
 final mockTransportCategory = ApiCategory(
   id: 2,
-  uuid: 'transport-uuid',
+  uuid: 'transports-uuid',
```

Comment:
👍 the plural!
- can you also fix other names like `mockAnimalsCategory ` & `mockTransportsCategory` ?